src/contexts/ListState.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { createContext, useContext, useReducer } from 'react'
```

Comment:
Just a question -- what is the reason for having a List Context state? I was under the impression that the "List" data would only ever be needed on the List page/component, but Context makes it available elsewhere. What is the advantage of having it as opposed to  just having a local state object in "ApplicationList"?

Reply:
As explained in person: When changing from one type to another the useListApplications hook was being called many times and the query for the previous type was re-running. So I decided to `reset` the list before querying for another one, and used the Context. I can investigate further why this was happening and try not to use the Context, I thought it would be useful to store other filters, but if you think we can simply use the local state and URL we can try that. By the way the Context was only available for the ApplicationsList, since that was the only Container with the Context provider wrapping it.

Reply:
Got it. I think it might be worth re-visiting this once we have all the filtering and everything else working, but all good for now.